Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
@wbond I'll make the change.  I preferred `keyword.operator` before we were scoping the identifier on the right (since it offered a strong visual distinction), but now that we're scoping the id, there's no real reason to avoid the punctuation scope.

Either way, I don't have much of a horse in the race since I don't use Java at all.  :-D